rotkehlchen/tasks/calendar.py

```diff
@@ -371,7 +371,7 @@ def maybe_create_airdrop_claim_reminder(self) -> None:
         data = check_airdrops(
             addresses=addresses,
             database=self.database,
-            data_dir=self.database.user_data_dir,
+            data_dir=self.database.user_data_dir.parent.parent,  # user_data_dir = self.data_directory / USERSDIR_NAME / username  # noqa: E501
```

Comment:
random thought. since `data_dir` is gotten from `self.database`, is there a need to pass it as an argument when `database=self.database` exists?

Reply:
It uses the data directory because it was already available where the function was used. I guess we can just move it
and use always .parent.parent
```
            data = check_airdrops(
                addresses=self.rotkehlchen.chains_aggregator.accounts.eth,
                database=self.rotkehlchen.data.db,
                data_dir=self.rotkehlchen.data.data_directory,
                tolerance_for_amount_check=AIRDROPS_TOLERANCE,
            )
```